glean_parser/templates/kotlin.jinja2

```diff
@@ -7,61 +7,61 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-{% macro metric_declaration(metric, suffix='', access='') %}
-{{ access }}val {{ metric.name|camelize }}{{ suffix }}: {{ metric|metric_type_name }} by lazy {
-        {{ metric|metric_type_name }}(
-            {% for arg_name in extra_args if metric[arg_name] is defined %}
-            {{ arg_name|camelize }} = {{ metric[arg_name]|kotlin }},
+{% macro obj_declaration(obj, suffix='', access='', lazy=False) %}
```

Comment:
nit: would it make sense to have 2 different templates? we're importing some things here that are not really useful for pings. What do you think?

Reply:
Given the complexity of the template just for the basics of generating Kotlin, I'd rather keep it to one template for everything.  Plus, pings aren't really a special case -- they are just another metric type as far as `glean_parser` is concerned.

Reply:
Got it, ok!